specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/CopyModel.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "body": {},
+    "copyRequest": {
+      "targetResourceId": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CognitiveServices/accounts/{resourceName}",
+      "targetResourceRegion": "{region}",
```

Comment:
will this work on dev endpoint ? Right now we are using the centraluseuap endpoint => would we have to put centraluseuap ? or westcentralus ? the targetEndpoint could be easier to provide

Reply:
FR currently supports only the eastus2euap canary region. So we will support Copy API in that region.